package.json

```diff
@@ -202,6 +204,7 @@
     "@typescript-eslint/parser": "^5.9.0",
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
+    "deep-equal-in-any-order": "^2.0.0",
```

Comment:
Looking at the way this is used, I think you should be able to omit this and do

```typescript
expect(Object.keys(obj)).to.have.members([/*...*/]);
```

https://www.chaijs.com/api/bdd/#method_members

LMK if that doesn't work for you.

Reply:
This isn't quite equivalent because `have.members` checks for set inclusion:

`expect([1,2,3]).to.have.members([1,2])`

The above evaluates to true whereas I want an exact match on the keys